src/ui/server/controllers/insurance/business/company-details/validation/company-details/rules/company-website.ts

```diff
@@ -0,0 +1,34 @@
+import { ERROR_MESSAGES } from '../../../../../../../content-strings';
+import { FIELD_IDS } from '../../../../../../../constants';
+import { RequestBody } from '../../../../../../../../types';
+
```

Comment:
Unnecessary space, linting probably flagging this

Reply:
done